attribution-data-file-share/src/main/java/gov/cms/ab2d/attributionDataShare/AttributionDataShare.java

```diff
@@ -0,0 +1,94 @@
+package gov.cms.ab2d.attributionDataShare;
+
+import gov.cms.ab2d.lambdalibs.lib.PropertiesUtil;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+
+public class AttributionDataShare {
+    
+    private static final String filePartialName = "ab2d-beneids_";
+    private static final String fileFormat = ".txt";
+
+    private static final String filePath = "/opt/";
+
+    private static final String SELECT_ALL_FROM_COVERAGE = "SELECT beneficiary_id from public.coverage";
+
+    // Returns a string with the fileFullPath to the file we wrote out.
+    // I.E: "ab2d-beneids_2023-08-16T12:08:56.235-0700.txt"
+    public String handleRequest(Context context) throws IOException {
+        LambdaLogger logger = context.getLogger();
+        logger.log("AttributionDataShare Lambda is started");
+
+        String currentDate = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ").format(new Date());
+
+        String fileFullName = filePartialName + currentDate + fileFormat;
+        
+        String fileFullPath = filePath + fileFullName;
+
+        try {
+            Connection dbConnection = getConnection();
+            List<String> coverageDataList = fetchCoverageData(dbConnection, SELECT_ALL_FROM_COVERAGE);
```

Comment:
what about OutOfMemoryError? 
this might be helpful https://github.com/CMSgov/ab2d/blob/master/coverage/src/main/java/gov/cms/ab2d/coverage/repository/CoverageServiceRepository.java#L531